app/uk/gov/hmrc/servicecommissioningstatus/persistence/CacheRepository.scala

```diff
@@ -67,13 +76,15 @@ object CacheRepository {
 
   case class ServiceCheck(
     serviceName: ServiceName
+  , status     : LifeCycleStatusRepository.LifeCycleStatusType
```

Comment:
call this param `serviceStatus` or `lifecycleStatus` - since `status` is ambiguous with ServiceCommissioningStatus.
Same for the json?